Description: lib/: Fix paths
 Upstream is aware of this patch and we're working on integrating it
 upstream.
Author: Georg Faerber <georg@riseup.net>
Last-Update: 2016-12-06
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/lib/schleuder/conf.rb
+++ b/lib/schleuder/conf.rb
@@ -91,7 +91,7 @@
     end
 
     def self.load_defaults(basename)
-      file = Pathname.new(ENV['SCHLEUDER_ROOT']).join("etc/#{basename}.yml")
+      file = Pathname.new(ENV['SCHLEUDER_CONFIG'])
       if ! file.readable?
         raise RuntimeError, "Error: '#{file}' is not a readable file."
       end
--- a/lib/schleuder/cli.rb
+++ b/lib/schleuder/cli.rb
@@ -66,7 +66,7 @@
     desc 'install', "Set-up or update Schleuder environment (create folders, copy files, fill the database)."
     def install
       config_dir = Pathname.new(ENV['SCHLEUDER_CONFIG']).dirname
-      root_dir = Pathname.new(ENV['SCHLEUDER_ROOT'])
+      root_dir = Pathname.new('/usr/lib/ruby/vendor_ruby/schleuder/')
 
       # Check if lists_dir contains v2-data.
       if Dir.glob("#{Conf.lists_dir}/*/*/members.conf").size > 0
--- a/lib/schleuder.rb
+++ b/lib/schleuder.rb
@@ -73,6 +73,7 @@
 end
 
 I18n.load_path += Dir["#{rootdir}/locales/*.yml"]
+I18n.load_path += Dir["/usr/share/schleuder/locales/*.yml"]
 I18n.enforce_available_locales = true
 I18n.default_locale = :en
 
